includes/class-pwa.php

```diff
@@ -25,6 +26,19 @@ public static function init() {
 
 		add_filter( 'wp_service_worker_navigation_caching', [ __CLASS__, 'increase_network_timeout' ] );
 		add_filter( 'wp_service_worker_error_messages', [ __CLASS__, 'error_messages' ] );
+
+		/**
+		 * Temporary workaround to disable the offline post request handling script.
+		 *
+		 * @see - https://github.com/GoogleChromeLabs/pwa-wp/issues/1106
+		 */
+		add_action(
+			'wp_front_service_worker',
+			function ( WP_Service_Worker_Scripts $scripts ) {
```

Comment:
This is a supreme nitpick, but let's stick with the prevailing pattern we tend to use for adding hook callbacks via static class methods.